container-conf/build.sh

```diff
@@ -8,3 +8,13 @@ build_as_root() {
     build_yum config-manager --set-enabled google-chrome
     build_yum install google-chrome-stable
 }
+
+build_as_run_user() {
+    umask 022
+    cd ~
+    git clone https://github.com/radiasoft/sirepo.git
```

Comment:
This is a reasonable way to do this, but it would be more efficient to: 

```sh
install_url radiasoft/sirepo
install_download package.json > package.json
```

install_url [knows about github.com](https://github.com/radiasoft/download/blob/84922815033e849b0f017102cb4e30572d98bd67/bin/install.sh#L391), which is the default install_server, and then it will download just the contents of package.json which is very small in comparison to the whole sirepo repo. 

install_download has [further hacks](https://github.com/radiasoft/download/blob/84922815033e849b0f017102cb4e30572d98bd67/bin/install.sh#L83) to know how to download efficiently from github.com, which is a bit quirky.

Reply:
One thought on the global install, if a PR has a different version specified (or adds/removes deps) then this won't pick it up. Is that a problem worth solving?

Reply:
This is a very narrow set of deps. The bigger problem is beamsim, on which this image is based. We will still manage that process separately. We release pretty rapidly so we don't need to worry about PRs that have dependency conflicts. The PR check will fail, and then we'll rebuild beamsim and sirepo-ci. Our goal is to make our software backwards/forwards compatible so the latest beamsim can run relatively recent sirepo code.

Reply:
91034b60c74e645b2b6c07a317c7888a4fa06831